reana-ui/src/pages/launchOnReana/Welcome.module.scss

```diff
@@ -12,15 +12,47 @@
   padding: 1em 0;
 }
 
+.card-container {
+  padding-bottom: 1em;
+}
+
 .demo-card {
-  .img-wrapper > a {
-    height: 200px;
-    display: flex;
-    justify-content: center;
-  }
+  margin-bottom: 2em;
 
   :global(.ui.button).btn:hover {
     color: white;
     background-color: $sui-blue;
   }
+
+  .img-container {
+    position: relative;
+    overflow: hidden;
+  }
+
+  // Add a pseudo element to keep the aspect ratio
+  .img-container::after {
+    content: "";
+    display: block;
+    padding-bottom: calc(100% / 1); // Define ratio here
+  }
+
+  img {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    object-fit: contain;
+  }
+
+  // Blur the background image and cover the whole container
+  // Use the negative blur value as offset to prevent the blur
+  // from mixing with the document background color
```

Comment:
What is the negative blur value?

Reply:
I had used it before as a way to prevent the blur from overflowing, but `overflow: hidden` did the trick better, so I've just deleted it.